### ISSUE TITLE
[Feature Request] Filter out outliers on the graph or ability to remove single data points from being graphed

### ISSUE THREAD

**Issue by IanVGO** [henrywhitaker3/Speedtest-Tracker#389]
**Describe the feature**
I'd love if there was a way to filter out the top and/or bottom 2-5% of results from the graph. A couple of outliers can cause the speed graph to zoom way out. Alternatively, the ability to remove single points of data from the web gui would also be a good feature.

**Additional context**
As you can see by the attached photo. There are a few points that are so far outside the usual range that they zoom the graph out. 
![image](https://user-images.githubusercontent.com/32964944/100114411-06608580-2e37-11eb-85a9-c127636fe02c.png)

**Comment by henrywhitaker3** [comment]
Yeah shouldn't be too hard to add this, am going to be working on it this weekend. Bit of a dirty workaround, but you could just delete the outliers in the meantime

**Comment by IanVGO** [comment]
Sounds good! If you end up not having time, I could take a crack at it this weekend too. I wanted to see if you would approve of the addition. Just let me know :)
